### PR TITLE
Circle & final mailroom

### PR DESCRIPTION
**Pull request by luftsmeerflier**
This PR includes the Circle exercise with unit testing as well as a draft of the final mailroom assignment.

### CODE REVIEW

File: students/luftsmeerflier/lesson08/circle.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		return self.radius > other.radius
+
+	def __ge__(self, other):
+		return self.radius >= other.radius
+
+	def __eq__(self, other):
+		return self.radius == other.radius
+
+	def __ne__(self, other):
+		return self.radius != other.radius
```
Comment (natasha-aleksandrova) [inline students/luftsmeerflier/lesson08/circle.py:50]: missing from_diameter() classmethod.

File: students/luftsmeerflier/lesson09/donor_models.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    def add_donation(self, name, donation, date):
+        if donation > 0 and name.replace(' ', '').isalpha():
+            if self.donors.get(name):
+                self.donors[name].add_donation(donation, date)
+            else:
+                self.donors[name] = Donor(name, donation, date)
+        elif donation < 0:
+            error_message = "The value '{}' for {} is unsuiable. Please remediate this".format(donation, name)
+            print(error_message)
+            exit()
```
Comment (natasha-aleksandrova) [inline students/luftsmeerflier/lesson09/donor_models.py:69]: you shouldn't have any `exit()` calls in your classes - these are useful for scripts.

**Review by natasha-aleksandrova — APPROVED** [review]
@luftsmeerflier great work overall - few minor comments.

**Comment by natasha-aleksandrova** [comment]
@luftsmeerflier a lot of your assignments are not submitted on the course website - we need to submit your grades in order for you to complete the course.

**Comment by luftsmeerflier** [comment]
HI Natasha,
The latest PR adds from_diameter() to the circle exercise and removes
exit() calls from the mailroom assignment in lesson09. I've also removed
the hard-coded donations so that the program inputs donors from scratch in
the main() loop.

I'll submit the remaining assignments as soon as I get this pr approved.

**Comment by natasha-aleksandrova** [comment]
sounds good 👍

**Comment by luftsmeerflier** [comment]
Hi Natasha,
I believe that my four month allotment is up for this course. Though I am
waiting in the PR approval. Will I still be able to submit the approved Pr
links to the submission platform in order to complete the course?

John

**Comment by natasha-aleksandrova** [comment]
John, I sent you an email about this. You need to submit all assignments on the course website as I need to submit your grade.

Please follow up with us via course email if there is any help you need.

**Pull request merged**